README.md

```diff
@@ -1,2 +1,29 @@
-# openmrs-frontend-spa
-The frontend SPA for OpenMRS code
+# openmrs-module-spa
+
+This project is an OpenMRS java module that makes it possible for Tomcat to serve an SPA instead of expecting all routes
+to be handled by server side JSPs/GSPs.
+
+It is part of the microfrotends architecture that is explained at https://github.com/openmrs/openmrs-rfc-frontend.
+
+## Development instructions
+1. Clone this repository.
+2. `mvn clean install`
+3. Copy `omod/target/spa-1.0.0-SNAPSHOT.jar` to the `~/.OpenMRS/modules` directory. On mac/linux, this is `cp omod/target/spa-1.0.0-SNAPSHOT.omod ~/.OpenMRS/modules`
+4. Add the following to [the web.xml file in openmrs-core](https://github.com/openmrs/openmrs-core/blob/master/webapp/src/main/webapp/WEB-INF/web.xml).
```

Comment:
Let's ask around about this (after merging).

Reply:
I think the suggestion from @djazayeri will maybe fix this. I'm going to try it out now. https://github.com/openmrs/openmrs-module-spa/pull/1#discussion_r283107536